### PR TITLE
Image data connection

### PR DESCRIPTION
**Pull request by erica-zhong**
This PR exposes ports that the websocket is created on for each server, so that the client is compatible with the newly dockerized server.

### CODE REVIEW

File: server/common/config.go [hunk 1]
```diff
@@ -3,5 +3,6 @@ package common
 type ServerConfig struct {
 	Hostname      string `json:"hostname"`
 	ApiPort       int    `json:"api_port"`
+	WebsocketPort int    `json:"websocket_port"`
```
Comment (erica-zhong) [inline server/common/config.go]: i can remove this line

File: client/src/constants.ts [hunk 2]
```diff
@@ -1 +1 @@
-export const HOSTNAME = '127.0.0.1:3010';
+export const HOSTNAME = '127.0.0.1:3000';
```
Comment (ThePatrickBrady) [inline client/src/constants.ts]: I forget why we changed this. @erica-zhong do you remember? Does it still work if we change it back?

File: server/apiserver/apiserver.go [hunk 3]
```diff
@@ -201,7 +201,8 @@ func reader(conn *websocket.Conn) {
 		}
 
 		// holds a map of strings to arbitrary data types
-		var dat map[string]interface{}
+		//var dat map[string]interface{}
```
Comment (ThePatrickBrady) [inline server/apiserver/apiserver.go]: remove old comment

File: docker-compose.yml [hunk 4]
```diff
@@ -4,6 +4,7 @@ services:
     image: owlplace
     ports:
       - "3001:3000"
+      - "3009:3000"
```
Comment (gvacaliuc) [inline docker-compose.yml]: why are we remapping another host port to 3000?

Comment (erica-zhong) [reply docker-compose.yml]: we were confused but now we are taking these lines out

File: server/apiserver/apiserver_test.go [hunk 5]
```diff
@@ -16,26 +20,60 @@ type handlerTest struct {
...
-func TestHello(t *testing.T) {
-	tests := []*handlerTest{
-		&handlerTest{
-			name:          "hello_test",
-			handler:       apiserver.Hello,
-			req:           &http.Request{},
-			recorder:      httptest.NewRecorder(),
-			expectWritten: []byte("hello\n"),
-		},
+// func TestHello(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (erica-zhong) [reply server/apiserver/apiserver_test.go]: yes

File: client/src/constants.ts (hunk 2 above)
Comment (erica-zhong) [reply client/src/constants.ts]: changing to 3001

**Pull request merged**